react.md

```diff
@@ -4,57 +4,61 @@ Try your best to answer each question on your own before looking up the answer o
 
 #### 1. Here is a list of pros and cons to using the React library to build your application -- but some of them are false. Remove the false statements from the list:
 
-- React was created to be simple, so that even people with minimal code experience could use it and create Single Page Applications (SPAs)
+- React was created to be simple, so that even people with minimal code experience could use it and create Single Page Applications (SPAs) (MAYBE????)
 - React is a modern, efficient answer to complex UI applications
-- React is a full stack framework for modern web applications
+
 - React is a flexible library that plays the role of V in an MVC framework
 
- 
+
  #### 2. What are "smart"(logic) and "dumb"(display) components? Explain the difference and also add why we bother to make the distinction between them.
- 
- 
+
+
  //Your Answer
- 
- 
+Smart components actually do logic and interact with the user. Dumb components simply display an element.
+
  //Googled Answer
- 
- 
+ Dumb (presentational/display) components are only responsible for presenting something to the dom. Smart components hold state to perform logic and make the app do things.
+
+
 #### 3. When we use "yarn add ..." in the terminal - what is yarn doing? And what file will always be automatically updated after we add a package with yarn?
- 
- 
+
+
  //Your Answer
- 
- 
+Yarn is a package manager. The yarn add ... will add the package  specified by the user (ex. "yarn add react" will add the files needed to run a react app).
+
  //Googled Answer
- 
- 
+ "Yarn add" installs a package and any packages that it depends on.
+
+
 #### 5. There are three mistakes in this code that would cause it to break our application. Find the mistakes and fix them:
 
-    import React, { Component } from 'react';
 
-    class Recipes {
+    import React, { Component } from 'react';
+                  //mistake 1
+    class Recipes extends Component {
       constructor(props){
         super(props)
         this.state = {
-          recipes: 
+          recipes:
```

Comment:
Nice work here! Hint: Looks like recipes in state is a list. What datatype is good for representing a list of items?